charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -150,16 +150,12 @@ model:
     z_threshold: 2.5
 
   prophet:
-    interval_width: 0.8
+    interval_width: 0.98
 #    country_holidays: 'US'
-#    seasonalities:
-#      - name: 'hourly'
-#        period: 0.04166666666
-#        fourier_order: 30
 
   holt_winters:
     seasonality: '1d'
-    frequency: '1h'   
+    frequency: '1m'
```

Comment:
this should be changed in master, as we still have `1h` there :+1: 
ideally, default frequency should be set equal to `sampling period` to follow the resolution of input data